src/app/services/ws.service.ts

```diff
@@ -59,19 +59,27 @@ export class WebSocketService {
           return throwError(() => data.error);
         }
 
-        // Mock Data
+        // Mock Data Response
+        const isEnabled = (
```

Comment:
Perhaps just exclude all of this to `canMockMessage()` function. Comments are unnecessary.

Reply:
Done